src/app/welcome/welcome.component.html

```diff
@@ -1,3 +1,7 @@
+<app-navbar [navBarItems]="[]"></app-navbar>
```

Comment:
we can add navbar component on app component template and show it when user is logged in and hide when not logged in. so no need to add it on each component separately.

Reply:
How will we identify which link to highlight in navbar if we put it in app component?

Reply:
After login, user should land on the same page, so on whichever page we navigate after login, we can check that route path to highlight sing routerLinkActive attribute

Reply:
Yeap got it

Reply:
Done, @akashP2312  check now